ndsl/comm/__init__.py

```diff
@@ -0,0 +1,15 @@
+from .boundary import SimpleBoundary
+from .caching_comm import (
+    CachingCommData,
+    CachingCommReader,
+    CachingCommWriter,
+    CachingRequestReader,
+    CachingRequestWriter,
+    NullRequest,
```

Comment:
Does NullRequest come up elsewhere/do we want to expose it?

Reply:
Type? not sure